SUVFactorCalculatorCLI/dcmHelpersCommon.cxx

```diff
@@ -20,7 +20,7 @@ const DcmTagKey dcmHelpersCommon::patientModuleTags[] = {
   DCM_QualityControlSubject,
   DCM_PatientBirthTime,
   DCM_ReferencedPatientSequence,
-  DCM_OtherPatientIDs,
+  DCM_RETIRED_OtherPatientIDs,
```

Comment:
Let's just remove this attribute from the list. See note in http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.2.2.html.